modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/display/context/ViewChangesDisplayContext.java

```diff
@@ -1476,6 +1476,11 @@ else if (_log.isDebugEnabled()) {
 							WorkflowConstants.STATUS_DRAFT)) {
 
 						modelInfo._jsonObject.put("showWorkflow", true);
+
+						modelInfo._jsonObject.put(
+							"showWorkflowActions",
+							_ctCollection.getStatus() ==
+								WorkflowConstants.STATUS_DRAFT);
```

Comment:
Can't we just apply this check to https://github.com/liferay/liferay-portal/blob/master/modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/action/GetEntryRenderDataMVCResourceCommand.java#L641-L646? The dropdown will not be rendered if `workflowActions` is null. That way we don't need to introduce a new prop.